server/server/management/commands/manage_token_restrictions.py

```diff
@@ -0,0 +1,128 @@
+from django.core.management.base import BaseCommand, CommandError
+from rest_framework.authtoken.models import Token
+from server.models import TokenIPRestriction
+from ipaddress import IPv4Network, IPv6Network, AddressValueError
+
+class Command(BaseCommand):
+    help = 'Manage IP restrictions for authentication tokens'
+
+    def add_arguments(self, parser):
+        parser.add_argument('token', help='Full token key to manage')
+        
+        # IP restriction actions
+        group = parser.add_mutually_exclusive_group(required=True)
+        group.add_argument('--add', help='Add an IP restriction (CIDR notation, e.g., 192.168.1.0/24)')
+        group.add_argument('--remove', help='Remove an IP restriction (CIDR notation)')
+        group.add_argument('--reset', action='store_true', help='Reset to default open policy (0.0.0.0/0 and ::/0)')
+        group.add_argument('--list', action='store_true', help='List current IP restrictions for this token')
+        
+        # Force option for potentially dangerous operations
+        parser.add_argument('--force', action='store_true', help='Force the operation without confirmation')
+
+    def validate_cidr(self, cidr):
+        """Validate CIDR notation"""
+        try:
+            IPv4Network(cidr)
+            return True
+        except (AddressValueError, ValueError):
+            try:
+                IPv6Network(cidr)
+                return True
+            except (AddressValueError, ValueError):
+                return False
```

Comment:
Since this is also needed by the `TokenIPRestriction` model, it should be moved there as a `staticmethod`